lib/Player.rb

```diff
@@ -0,0 +1,10 @@
+class Player
+attr_reader :name, :move
+  def initialize(args)
+    @name = args
+  end
+
+  def choice(args)
```

Comment:
The name makes it sound like it's going to return a choice but actually it's recording a choice. Using `args` as a parameter name is also not very descriptive (here and above).

```suggestion
  def record_choice(choice)
```